dojo/jira_link/helper.py

```diff
@@ -476,14 +476,22 @@ def log_jira_message(text, finding):
 def get_labels(obj):
     # Update Label with system setttings label
     labels = []
+
+    # Put a branch tag in as a label.
+    if type(obj) == Finding and obj.test.branch_tag:
+        labels.append(obj.test.branch_tag.replace(" ", "_"))
+
     system_settings = System_Settings.objects.get()
     system_labels = system_settings.jira_labels
-    if system_labels:
+    if system_labels is None:
+        return labels
+    else:
         system_labels = system_labels.split()
+    if len(system_labels) > 0:
         for system_label in system_labels:
             labels.append(system_label)
-        # Update the label with the product name (underscore)
-        labels.append(prod_name(obj).replace(" ", "_"))
+    # Update the label with the product name (underscore)
+    labels.append(prod_name(obj).replace(" ", "_"))
```

Comment:
this is skipped if there are no system labels, might be better to always add it

Reply:
Sure Valentijn. But what I found that with this code change either the labelling is done automatically if the report contains the branch information or else while importing new report if we mention branch tag, the labels are added for all those findings in Jira which was not possible with the default helper.py code. With the default code only in description (field containing branch ) tagging was happening.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/37924175/151121320-77b66b9e-d379-4409-aff6-710cd023ffc7.png">

Reply:
I understand, but my comment is about the product name not being added to the labels if there are no system labels. I think product name should always be added as label, as that was the intended behaviour I think. Currently this also doesn't happen, but if we are working on this code we might as well fix that.

Reply:
Sure will work with one of my colleague to fix this.